packages/core/karma.conf.js

```diff
@@ -10,7 +10,12 @@ module.exports = function (config) {
     const baseConfig = createKarmaConfig({
         dirname: __dirname,
         coverageExcludes: [
+            // not worth full coverage
             "src/accessibility/*",
+            "src/common/abstractComponent*",
+            // deprecated components
+            "src/components/popover/*",
```

Comment:
are these actually deprecated if they're being replaced by labs components?

Reply:
we'll need to remember to remove this line when @cmslewis's Popover2 -> Popover change goes.